dask/bag/core.py

```diff
@@ -182,7 +183,8 @@ def to_textfiles(b, path, name_function=str):
     return Bag(merge(b.dask, dsk), name, b.npartitions)
 
 
-class Item(object):
+class Item(DaskBase):
+    _optimize = optimize
```

Comment:
I like this idea.  `dask.imperative.value` should consider calling this whenever it reaches another dask base object.